app/controllers/suggestions.rb

```diff
@@ -0,0 +1,75 @@
+get '/suggestions' do
+  @suggestions = Suggestion.all
+  @suggestions_by_votecount = @suggestions.sort_by{|suggestion| suggestion.votes.count}.reverse
+  erb :'suggestions/index'
+end
+
+get '/suggestions/new' do
+  @suggestion = Suggestion.new
+  erb :'suggestions/new'
+end
+
+post '/suggestions' do
+  @suggestion = current_user.suggestions.create(params[:suggestion])
```

Comment:
👍  leveraging `User#suggestions` to associate the new suggestion with the authenticated user

Reply:
Next would be to implement error messaging in case this create fails.